### DISCUSSION TITLE
💡 Feature Request: More info on /location, or other alternative

### DISCUSSION THREAD

**Discussion by Dx8pi** [DiscordDungeons/Bugs#1329] [Ideas]
### Is your feature request related to a problem?

Oftentimes, locations have more info to them thsn what says in /location, such as area-specific sides loot, or area-specific mobs that have mob-spcific drops.

### Describe alternatives you've considered

I suggest a more comprehensive menu/second page that contains all info on items you can get from that specific location. Maybe even more, like quest starts and whatnot. ~~will also help me make maps cause I get more lore lmao~~

**Comment by FelixArgyle** [comment]
Already have that up at, https://github.com/DiscordDungeons/Bugs/discussions/951 
https://github.com/DiscordDungeons/Bugs/discussions/953 (well that mob / level specific, but can just change title and comment more ofc.

also pepega mackan not told you to use the discussions tab rather than (issues) since issues is for bug, discussions have suggestions / accepted suggestions all at https://github.com/DiscordDungeons/Bugs/discussions
